src/main/steps/c100-rebuild/safety-concerns/abduction/child-location/template.njk

```diff
@@ -0,0 +1,20 @@
+{% extends "common/template.njk" %}
+
+{% block page_content %}
+
+    <span class="govuk-caption-l">{{section}}</span>
+    <h1 class="govuk-heading-xl">{{title}}</h1>
+    <div class="govuk-warning-text">
```

Comment:
@ShakiraKP We don't need write all this code, there is a macro available in the framework.
We can simply a define an array in content.ts & use macro in template.njk to render.
Please refer to hearing-urgency/urgent for reference
`{{ govukWarningText(getContent(warningText)) }}`

Reply:
Oh okay, sorry I forgot this advice and used an example I found, will adjust. @channa-patil

Reply:
@ShakiraKP Thanks.
Please mark Resolve only after you pushed the fixes.

Reply:
Completed